### PR TITLE
Adding Tourniquets for bleeds

### PR DESCRIPTION
**Pull request by PoorMansDreams**
## About the PR

Adds a very fast acting Tourniquet to stop bleeding, but does some brute/aphyx damage in turn.
## Why / Balance

Gauze and Tranx needles often take too long and you'll probably bleed out before fully injecting it
## Media

https://www.youtube.com/watch?v=d_qTfsNhlSo
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

- add: Added Tourniquets, find them in your local Sectechs.

### CODE REVIEW

**Comment by PoorMansDreams** [comment]
I fucked up bad

**Comment by deltanedas** [comment]
i dont think this should be a thing until wounds, since irl putting on tourniquet is bad

File: Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml [hunk 1]
```diff
@@ -225,6 +225,32 @@
   - type: Stack
     lingering: true
 
+- type: entity
+  parent: BaseHealingItem
+  id: Tourniquet
+  name: Tourniquet
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:231]: AHHHHH MY EYES, NAME FROM CAPITAL LETTER AHHHH

Comment (PoorMansDreams) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml:231]: time to fix

**Comment by Aisu9** [comment]
for information bleedrate is capped at 10, because people cannot deal with heavy bleeding

**Comment by PoorMansDreams** [comment]
nevermind horrors not fixed I will be bac

**Comment by PoorMansDreams** [comment]
im just gonna close this and open another becuase I royally fucked my git

**Pull request closed without merging**